### PR TITLE
Sanity test / doc-default-does-not-match-spec fixups

### PR DESCRIPTION
**Pull request by tremble**
##### SUMMARY

Daily run picked up the following:
```
plugins/modules/aws_s3.py:0:0: doc-default-does-not-match-spec: Argument 'ignore_nonexistent_bucket' in argument_spec defines default as (False) but documentation defines default as (None)
plugins/modules/ec2.py:0:0: doc-default-does-not-match-spec: Argument 'termination_protection' in argument_spec defines default as (None) but documentation defines default as (False)
plugins/modules/ec2_elb_lb.py:0:0: doc-default-does-not-match-spec: Argument 'cross_az_load_balancing' in argument_spec defines default as (None) but documentation defines default as (False)
plugins/modules/ec2_vpc_dhcp_option_info.py:0:0: doc-default-does-not-match-spec: Argument 'dry_run' in argument_spec defines default as (False) but documentation defines default as (None)
plugins/modules/s3_bucket.py:0:0: doc-default-does-not-match-spec: Argument 'ceph' in argument_spec defines default as (False) but documentation defines default as (None)
```

Some additional doc-style fixups.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

plugins/modules/aws_s3.py
plugins/modules/ec2.py
plugins/modules/ec2_elb_lb.py
plugins/modules/ec2_vpc_dhcp_option_info.py
plugins/modules/s3_bucket.py

### CODE REVIEW

File: plugins/modules/aws_s3.py [hunk 1]
```diff
@@ -94,10 +94,12 @@
   overwrite:
     description:
       - Force overwrite either locally on the filesystem or remotely with the object/key. Used with PUT and GET operations.
-        Boolean or one of [always, never, different], true is equal to 'always' and false is equal to 'never', new in 2.0.
-        When this is set to 'different', the md5 sum of the local file is compared with the 'ETag' of the object/key in S3.
+      - Must be a Boolean, C(always), C(never) or C(different).
+      - C(true) is the same as C(always).
+      - C(false) is equal to C(never)
```
Comment (felixfontein) [inline plugins/modules/aws_s3.py:99]:
```suggestion
      - C(false) is equal to C(never).
```

File: plugins/modules/ec2.py [hunk 2]
```diff
@@ -111,7 +111,7 @@
     type: int
   monitoring:
     description:
-      - Enable detailed monitoring (CloudWatch) for instance.
+      - Enable detailed monitoring (CloudWatch for instance).
```
Comment (felixfontein) [inline plugins/modules/ec2.py:114]: Maybe the original version was correct, when the ec2 instance was meant.

File: plugins/modules/ec2.py [hunk 3]
```diff
@@ -156,8 +159,8 @@
   termination_protection:
     description:
       - Enable or Disable the Termination Protection.
+      - Defaults to C(False)
```
Comment (felixfontein) [inline plugins/modules/ec2.py:162]:
```suggestion
      - Defaults to C(false).
```

File: plugins/modules/ec2.py (hunk 2 above)
Comment (tremble) [reply plugins/modules/ec2.py:114]:
```suggestion
      - Enable detailed monitoring (CloudWatch) for the instance.
```

I think you're right, I've made a slight tweak: the missing article made my mind parse it a little oddly.

**Review by felixfontein — APPROVED** [review]

**Pull request merged**